api/application_submissions/index.js

```diff
@@ -31,24 +31,31 @@ router.post('/', async (req, res) => {
   }
 });
 
-router.put('/:id', async (req, res) => {
-  const { id } = req.params;
+router.put('/', async (req, res) => {
+  const { ids } = req.body; 
   const { decision } = req.body;
 
   try {
-    let applicationSubmission = await ApplicationSubmission.findById(id);
-    if (applicationSubmission) {
-      if (decision === 'ACCEPTED') {
-        const { skilled_impact_request_id } = applicationSubmission;
-        applicationSubmission = await ApplicationSubmission.acceptAndDenyAllOthers(id, skilled_impact_request_id);
-      } else {
-        applicationSubmission = await ApplicationSubmission.update(id, decision);
-      }
-      res.status(200).json({ applicationSubmission, message: 'Submission updated in database' });
-    } else {
-      res.status(404).json({ message: 'Submission not found in database' });
+    const applicationSubmissions = await ApplicationSubmission.updateAll(ids, decision);
+    // return 200 if all submissions successfully updated
+    if(applicationSubmissions.length == ids.length) {
```

Comment:
Triple equals please

Reply:
Not sure if I can reply to your original comment thread but yes you are correct. This change allows the organization to accept multiple people without automatically rejecting everyone else after only one person has been accepted. The org will decide how many people to accept, and once they are satisfied, they can select everyone else to reject.